Overall edit of PyPDF-Core files.

PipKat opened the pull request:
Edits to example files were limited to those necessary to address style issues discovered by Vale. All content will need to be reviewed again after completion of the PR. Note that some diagrams in examples were not being generated (locally at least), and lots of formatting changes are necessary to the examples.

PipKat commented:
@PProfizi Lots of "Build and Test" checks failing.

PProfizi commented:
> @PProfizi Lots of "Build and Test" checks failing.

Hi @PipKat, yes this is normal, we are in the process of releasing PyPDPF-Core 0.6.0 due to a milestone, which means we had to release higher versions of its dependencies, potentially breaking the pipelines. The release should be done within two hours and everything should be working again.

PipKat commented:
@RobPasMue I don't know how these example files got into this PR. I swear it wasn't like this on Friday. It was just failing Build and Test checks.

RobPasMue commented:
Hi @PipKat from what I can see, the files where already there. And by updating the examples they got also updated... Probably @PProfizi can tell us a bit more. In any case, the examples files shouldn't be in the ``docs/source/examples/*`` directory. They should be in the ``examples`` folder at the root of the repository. These are all auto-generated files.

PProfizi commented:
@PipKat @RobPasMue I am not sure when this got here, apparently there might be a commit missing somewhere where they were removed, but I'll remove it.

PipKat commented:
@PProfizi @RobPasMue  YES! All checks have passed. Now for a review?

PProfizi reviewed (changes requested):
I think the docs/source/index.rst got mixed up as it is now made as if referring to PyDPF-Post.

PipKat commented on docs/source/user_guide/waysofusing.rst:
```diff
@@ -1,21 +1,26 @@
-custom results plots using numpy and matplotlib libraries.
+CPython
+-------
+Standalone DPF uses CPython and can be accessed via any Python console. 
```
```suggestion
Standalone DPF uses CPython and can be accessed with any Python console. 
```

PipKat commented on docs/source/user_guide/waysofusing.rst:
```diff
@@ -1,21 +1,26 @@
-``DPF in Mechanical`` uses IronPython and is accessible via the ACT console. 
+IronPython
+----------
+DPF in Mechanical uses IronPython and is accessible via the **ACT Console**. 
```
```suggestion
DPF in Mechanical uses IronPython and is accessible with the **ACT Console**. 
```

PipKat commented on examples/07-python-operators/02-python_operators_with_dependencies.py:
```diff
@@ -133,19 +142,21 @@
 from ansys.dpf.core import examples
 
-# python plugins are not supported in process
+# Python plugins are not supported in process
```
```suggestion
# Python plugins are not supported in process.
```

PipKat commented:
@PProfizi Hi, Paul. I'm sure I introduced those problems somehow when I pulled in changes from the main branch. I ended up with conflicts in these files. I'm not sure what I did wrong since it seems so long ago that I took that action, but thanks for catching the issues. I've incorporated your feedback to date and hope that you can find time to finish your review and approve the PR. Thanks!

PipKat commented:
@PProfizi The Build and Test (3.10, ubuntu-latest check has been running the entire day!
